Polyhedron/demo/Polyhedron/Plugins/PMP/CMakeLists.txt

```diff
@@ -90,10 +111,14 @@ target_link_libraries(isotropic_remeshing_plugin  scene_polyhedron_item scene_po
 polyhedron_demo_plugin(distance_plugin Distance_plugin)
 target_link_libraries(distance_plugin scene_polyhedron_item scene_color_ramp)
 
+
 polyhedron_demo_plugin(detect_sharp_edges_plugin Detect_sharp_edges_plugin)
 target_link_libraries(detect_sharp_edges_plugin scene_polyhedron_item)
 
 polyhedron_demo_plugin(detect_sharp_edges_sm_plugin Detect_sharp_edges_plugin)
 target_link_libraries(detect_sharp_edges_sm_plugin scene_surface_mesh_item)
 target_compile_definitions(detect_sharp_edges_sm_plugin PUBLIC "-DUSE_SURFACE_MESH" )
+polyhedron_demo_plugin(distance_plugin Distance_plugin)
+target_link_libraries(distance_plugin scene_polyhedron_item  scene_color_ramp)
+target_compile_definitions(distance_sm_plugin PUBLIC "-DUSE_SURFACE_MESH" )
```

Comment:
This plugin `distance_plugin` is already defined a few lines above!

I remove this PR from `integration`...